### PR TITLE
Add service setting to change annual limits for platform admins

### PR DESCRIPTION
**Pull request by whabanks**
# Summary | Résumé

This PR adds capability for platform admins to set annual email and sms limits for a service. It also changes the SMS and Email annual limit displays so they are no longer hard coded but fetched from the DB.

# Test instructions | Instructions pour tester la modification
All `service-{id}` keys need to be cleared when deployed to staging / prod

1. If it is not yet merged, check out [this API PR](https://github.com/cds-snc/notification-api/pull/2329/files#diff-02a4832ab85a57344c4179cf331c533fa2bf4367833127a54e479dcf986a66d3) and run `flask db upgrade`
2. Login as a platform admin
3. Navigate to a service's settings
4. In the platform admin section, modify both the email and SMS annual limit
5. Note that the limits are reflected in the email and text message sections above

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -239,6 +261,13 @@ class ProductionFF(Config):
     NO_BRANDING_ID = "0af93cf1-2c49-485f-878f-f3e662e651ef"
 
     FF_RTL = False
+    FF_SPIKE_SMS_DAILY_LIMIT = True
+    FF_SMS_PARTS_UI = False
+    FF_BOUNCE_RATE_V1 = True
+    FF_BOUNCE_RATE_V15 = True
+    FF_BOUNCE_RATE_BACKEND = True
+    FF_ABTEST_SERVICE_ID = ""
```
Comment (andrewleith) [inline app/config.py:269]: These should be removed as they are no longer in use.

File: app/config.py [hunk 2]
```diff
@@ -239,6 +261,13 @@ class ProductionFF(Config):
...
+    FF_ANNUAL_LIMIT = True
```
Comment (andrewleith) [inline app/config.py:270]: This section is to mimic what values we will use in prod.  I assume we will release this with `FF_ANNUAL_LIMIT` off, so this should be set to `False` for now.

**Review by andrewleith — APPROVED** [review]
LGTM in general, works as expected with the API PR. 

Just the nagging issue of the formatting but that doesnt impact this functionally.

File: app/config.py (hunk 1 above)
Comment (whabanks) [reply app/config.py]: Fixed!

File: app/config.py (hunk 2 above)
Comment (whabanks) [reply app/config.py]: Fixed!

**Pull request merged**